src/Config.js

```diff
@@ -71,6 +71,22 @@ class Config {
   static get maxRecordsForQALookups() {
     return process.env.MAX_RECORDS_FOR_QA_LOOKUPS || 8
   }
+
```

Comment:
For the MVP I think it is fine to hard-code these values; we'll need to add all 17 members of the cohort in this list. In future if we have 100s of different institutions, this method seems not scalable. Not sure what the best method for this, maybe an Elasticsearch query when we have the RDF data?  Not blocking.

Reply:
Maybe a trellis api call to `/repository` using https://github.com/LD4P/sinopia_editor/blob/ff314f1b36015b7b4a927e7eabb1e3e914c321e5/src/sinopiaServer.js#L33 ? Initially adding the groups would be handled via `sinopia_acl`